modals/src/main/java/com/decathlon/vitamin/compose/modals/impl/ModalButtons.kt

```diff
@@ -0,0 +1,78 @@
+package com.decathlon.vitamin.compose.modals.impl
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.Dp
+import com.decathlon.vitamin.compose.modals.ModalButtonsOrientation
+
+internal object ModalButtons {
+    private const val MAX_BUTTONS_COUNT = 3
+
+    @Composable
+    internal fun Buttons(
+        modifier: Modifier,
+        buttons: List<(@Composable () -> Unit)>,
+        buttonsOrientation: ModalButtonsOrientation,
+        topPadding: Dp,
+        buttonsSpacer: Dp
+    ) {
+        if (buttons.isNotEmpty()) {
+            if (buttons.size > MAX_BUTTONS_COUNT) {
+                throw IllegalArgumentException("VitaminModals does not accept more than 3 buttons")
+            }
+
+            if (buttonsOrientation == ModalButtonsOrientation.HORIZONTAL) {
+                Row(
+                    modifier = modifier
+                        .fillMaxWidth()
+                        .padding(top = topPadding),
+                    horizontalArrangement = Arrangement.End
+                ) {
+                    buttons.getOrNull(2)?.invoke()
```

Comment:
I'm not sure to understand the need to change from named parameter vs list parameter.
You can do the exact same implementation with the old contract, can you explain why you suggest to change?